mParticle-Primer.podspec

```diff
@@ -1,20 +1,20 @@
 Pod::Spec.new do |s|
-    s.name             = "mParticle-CompanyName"
+    s.name             = "mParticle-Primer"
     s.version          = "6.2.0"
-    s.summary          = "CompanyName integration for mParticle"
+    s.summary          = "Primer integration for mParticle"
 
     s.description      = <<-DESC
-                       This is the CompanyName integration for mParticle.
+                       This is the Primer integration for mParticle.
                        DESC
 
     s.homepage         = "https://www.mparticle.com"
     s.license          = { :type => 'Apache 2.0', :file => 'LICENSE' }
     s.author           = { "mParticle" => "support@mparticle.com" }
-    s.source           = { :git => "https://github.com/mparticle-integrations/mparticle-apple-integration-companyname.git", :tag => s.version.to_s }
+    s.source           = { :git => "https://github.com/mparticle-integrations/mparticle-apple-integration-primer.git", :tag => s.version.to_s }
     s.social_media_url = "https://twitter.com/mparticles"
 
-    s.ios.deployment_target = "7.0"
-    s.ios.source_files      = 'mParticle-CompanyName/*.{h,m,mm}'
+    s.ios.deployment_target = "8.0"
+    s.ios.source_files      = 'mParticle-Primer/*.{h,m,mm}'
     s.ios.dependency 'mParticle-Apple-SDK/mParticle', '~> 6.2'
-    #s.ios.dependency 'CompanyName', '9.9.9'
+    s.ios.dependency 'Primer', '~> 3.1'
```

Comment:
Is it possible to pin the version to a specific number? We update versions frequently, and this has been a request we received from clients where they have tested and certified a version for release only to have to redo the job because a pod updated on them.